### PR TITLE
RSC bots phase II - Csharp and nodejs

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Updated Manifest of sample to support chat message support.

### CODE REVIEW

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, approving it.

File: samples/bot-receive-channel-messages-withRSC/csharp/README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ This feature shown in this sample is currently available in Public Developer Pre
 
 5. This step is specific to Teams.
     - **Edit** the `manifest.json` contained in the  `AppManifest` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`) also update the `<<DOMAIN-NAME>>` with the ngrok URL`
+    - **Edit** the `manifest.json` for `<<MANIFEST-ID>>` replace it with any `GUID` using guid generator - [Guid Generator](https://guidgenerator.com/online-guid-generator.aspx).
```
Comment (Wajeed-msft) [inline samples/bot-receive-channel-messages-withRSC/csharp/README.md:80]: Not sure how long the referenced tool would be available. We can keep the existing GUID in manifest.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
**Executed build pipeline:**
**Result:**

**Pull request merged**